databuilder/rcpai/base_arg_parser.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+from dataclasses import dataclass
+from typing import Any
+
+from elasticsearch import Elasticsearch
+
+DEV_HOST = "10.142.20.66"
+
+
+@dataclass
+class Neo4JConf(object):
+    endpoint: str
+    user: str
+    password: str
+    encrypted: bool = False
+
+
+class RCPArgParser(object):
```

Comment:
There was also a lot of shared argument parsing, which I have now extracted to this class.